docs/workload-tutorial/hello_world/stage_3/CMakeLists.txt

```diff
@@ -14,6 +14,9 @@
 
 CMAKE_MINIMUM_REQUIRED(VERSION 3.2 FATAL_ERROR)
 
+# Common components for both trusted and untrusted common libraries
```

Comment:
Can we remove the comment since its not a common library.

Reply:
Removed.